nilearn/surface/surface.py

```diff
@@ -582,14 +636,23 @@ def vol_to_surf(
         The size (in mm) of the neighbourhood from which samples are drawn
         around each node. Ignored if `inner_mesh` is provided.
 
-    interpolation : {'linear', 'nearest'}, default='linear'
+    interpolation : {'linear', 'nearest', 'nearest_most_frequent'}, \
+                    default='linear'
         How the image intensity is measured at a sample point.
 
         - 'linear':
             Use a trilinear interpolation of neighboring voxels.
         - 'nearest':
             Use the intensity of the nearest voxel.
 
+        .. versionadded:: 0.11.2.dev
+
+        - 'nearest_most_frequent':
+            Use the most frequent value in the neighborhood (out of the
+            `n_samples` samples) instead of the mean value. This is useful
+            when the image is a
+            :term:`deterministic atlas<Deterministic atlas>`.
+
```

Comment:
```suggestion
        - 'nearest_most_frequent':
            Use the most frequent value in the neighborhood (out of the
            `n_samples` samples) instead of the mean value. This is useful
            when the image is a
            :term:`deterministic atlas<Deterministic atlas>`.
            
            .. versionadded:: 0.11.2.dev

```

Reply:
I think you want the directive after the new option.

Reply:
Ok yes thanks!